src/addons/transitions/__tests__/ReactTransitionGroup-test.js

```diff
@@ -128,7 +126,7 @@ describe('ReactTransitionGroup', function() {
       },
       render: function() {
         var children = [];
-        for (var i = 0; i < this.state.count; i++) {
+        for (let i = 0; i < this.state.count; i++) {
```

Comment:
We already don't run the tests internally. But we still lint internally and I wouldn't be surprised if we end up choking on let. Regardless though, let's not introduce the first `let` here.